OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyFinalFragment.java

```diff
@@ -328,6 +329,9 @@ public SaveKeyringParcel createOperationInput() {
             @Override
             public void onCryptoOperationSuccess(EditKeyResult result) {
 
+                BackupNotificationUtil.setBackupReminder(getContext(),result.mMasterKeyId,
```

Comment:
You can also do it before `finishWithResult`, the important thing is that it's after the `null` check.